lisa/tools/tlb_stress.py

```diff
@@ -0,0 +1,1036 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Move this file to the test suite folder since it's not a standalone tool, and it points to the test suite folder.

Reply:
done